src/containers/survey/SurveyContainer.js

```diff
@@ -58,9 +67,16 @@ const SurveyContainer = () => {
   } :{ date :string, organizationId :UUID, participantId :string, studyId :UUID } = queryParams;
 
   // selectors
-  const userAppsData = useSelector((state) => state.getIn(['appsData', 'appsData'], Map()));
-  const getUserAppsRS :?RequestState = useRequestState(['appsData', GET_CHRONICLE_APPS_DATA]);
-  const submitSurveyRS :?RequestState = useRequestState(['appsData', SUBMIT_SURVEY]);
+  const userAppsData = useSelector((state) => state.getIn([REDUCERS.APPS_DATA, 'appsData'], Map()));
+  const settings = useSelector((state) => state.getIn([REDUCERS.APP, SETTINGS], Map()));
```

Comment:
if you think you might use these again, might be worth pulling out into a separate file.